cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/index.js

```diff
@@ -0,0 +1,67 @@
+// import { render } from 'react-dom';
+
+// const App = () => (
+//     <section className="my-money-calendar">
+//      <div>hi, missy app</div>
+//     </section>
+// );
+
+// render(<App />, document.querySelector('#mmt-cost-of-credit-tool'));
+
+import * as idb from "idb";
+import { render } from "react-dom";
+import { Workbox } from "workbox-window";
+import { DateTime, Info } from "luxon";
+
+import Routes from "./routes";
+
+// configureMobX({ enforceActions: 'observed' });
```

Comment:
Remove everything you're not using. This app is probably not using MobX, and it has nowhere near the complexity of the other one. You can keep things simple and pull stuff from the other app's codebase if the need arises.

Reply:
done.